IPs/RSKIP148.md

```diff
@@ -0,0 +1,906 @@
+# Bridge events data and formatting
+
+|RSKIP          |148           |
+| :------------ |:-------------|
+|**Title**      |ERC1820 Pseudo-introspection Registry Contract|
+|**Created**    |6-NOV-19 |
+|**Author**     |PMP |
+|**Purpose**    |Usa |
+|**Layer**      |DApp|
+|**Complexity** |1 |
+|**Status**     |Adopted |
+
+
+## Abstract
+
+This RSKIP proposes complying with the ERC1820 specification, and deployment at he address of the registry on all chains 0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24 as a standard interface detection.
```

Comment:
just fix this typo

"and deployment at he address.."

Reply:
fixed